src/app/popular/layout.js

```diff
@@ -0,0 +1,7 @@
+import Page from "./page";
+
+export const metadata = {
+    title: "Popular Repositories - RepoRater",
+};
+
+export default Page;
```

Comment:
This needs a blank line at the end, as indicted by GitHub with the red icon

```suggestion
export default Page;

```